Index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>benjaminSemah.com</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav class="logo">
+            <a class="myLogo" href="#">My Logo</a>
+        </nav>
+        <div class="navIcon">
+            <a href="#"><img src="assets/navIcon.svg" alt="navigation icon"></a>
+        </div>
```

Comment:
- [ ] I think what the previous reviewer suggested was to use the `<nav>` tag for the image since this is the image for the "navigation" links. I would suggest updating this so your structure looks like this :raised_hands:
```
<header>
  <a class="myLogo" href="#">My Logo</a>
  <nav class="navIcon">
    <a href="#"><img src="assets/navIcon.svg" alt="navigation icon"></a>
  </nav>
</header>
```